src/general/OptionsMenu.tscn

```diff
@@ -521,29 +524,95 @@ __meta__ = {
 }
 
 [node name="Intro" type="CheckBox" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer"]
-margin_right = 502.0
+margin_right = 188.0
 margin_bottom = 25.0
 size_flags_horizontal = 0
 custom_styles/hover_pressed = SubResource( 2 )
 text = "Play intro video"
 
 [node name="MicrobeIntro" type="CheckBox" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer"]
 margin_top = 35.0
-margin_right = 502.0
+margin_right = 368.0
 margin_bottom = 60.0
+rect_pivot_offset = Vector2( 114, 23 )
 size_flags_horizontal = 0
 custom_styles/hover_pressed = SubResource( 2 )
 text = "Play Microbe Intro on New Game"
 
-[node name="HSeparator" type="HSeparator" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer"]
+[node name="AutoSave" type="CheckBox" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer"]
 margin_top = 70.0
-margin_right = 502.0
-margin_bottom = 74.0
+margin_right = 307.0
+margin_bottom = 95.0
+rect_pivot_offset = Vector2( 114, 23 )
+size_flags_horizontal = 0
+custom_styles/hover_pressed = SubResource( 2 )
+text = "Autosave during the game"
+
+[node name="HBoxContainer" type="HBoxContainer" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer"]
+margin_top = 105.0
+margin_right = 520.0
+margin_bottom = 130.0
+
+[node name="Label" type="Label" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer/HBoxContainer"]
+margin_top = 1.0
+margin_right = 301.0
+margin_bottom = 24.0
+text = "Amount of autosaves to keep:"
+
+[node name="HSeparator2" type="HSeparator" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer/HBoxContainer"]
+margin_left = 305.0
+margin_right = 316.0
+margin_bottom = 25.0
+size_flags_horizontal = 3
+custom_styles/separator = SubResource( 5 )
+
+[node name="MaxAutoSaves" type="OptionButton" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/Misc/VBoxContainer/HBoxContainer"]
```

Comment:
I think that it makes more sense to use a SpinBox for this with something like minimum value of 1 (unchecking the enabled button should also make the spinbox inactive to tell the user that that's how to select 0), and maximum of 50. The same for quick saves.

Reply:
I did originally try a SpinBox, but it didn't seem to be themed so I went with what had already been used before.

Reply:
It can be themed later (if you don't want to do it). I've also used a bunch of controls that don't have themes on them yet.